src/workspaces/infrastructure/persistence/subscibers/workspace.subscriber.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  EntitySubscriberInterface,
+  EventSubscriber,
+  InsertEvent,
+} from 'typeorm';
+
+import { WorkspaceEntity } from '../entities/workspace.entity';
+
+@EventSubscriber()
+export class MessageSubscriber
+  implements EntitySubscriberInterface<WorkspaceEntity>
+{
+  listenTo() {
+    return WorkspaceEntity;
+  }
+
+  async afterInsert(event: InsertEvent<WorkspaceEntity>) {
+    const channal = {
+      title: 'general',
+      description: 'This is a general channal',
```

Comment:
This is the one channel that will always include everyone. It’s a great spot for announcements and team-wide conversations.